modules/tfenv/manifests/init.pp

```diff
@@ -0,0 +1,65 @@
+# == Class: tfenv
+#
+class tfenv(
+  $install_dir          = '/opt/tfenv',
+  $tfenv_git_repo       = 'https://github.com/kamatama41/tfenv',
```

Comment:
Should this be
```suggestion
  $tfenv_git_repo       = 'https://github.com/tfutils/tfenv',
```

? Looks like it redirects.

Reply:
Yep, can fix that, we copied across the code contents of the puppet module: https://forge.puppet.com/modules/sergk/tfenv because of some dependencies conflict